manticore/core/state.py

```diff
@@ -3,7 +3,7 @@
 import logging
 from collections import OrderedDict
 
-from .smtlib import solver
+from .smtlib import solver, Bool
```

Comment:
Bool was always used, it just stopped being imported for a bit (https://github.com/trailofbits/manticore/blob/42b462fc6c2c9ee66393991d692215475b09fcd4/manticore/core/state.py#L58)